fieldservice/views/fsm_order.xml

```diff
@@ -21,7 +21,9 @@
                             attrs="{'invisible': [('stage_id', 'in', (%(fieldservice.fsm_stage_completed)d, %(fieldservice.fsm_stage_cancelled)d))]}"/>
                     <field name="stage_id" widget="statusbar"
                            options="{'fold_field': 'fold'}"
-                           domain="[('stage_type', '=', 'order')]"/>
+                           domain="['&amp;', '|',
```

Comment:
You can shorten it to : (stage_type', '=', 'order'), ('team_ids', 'in', (team_id', False))